### PR TITLE
increase field length for procedure code in procedure_type, remove unwanted contrib file

### LINKED ISSUE #5351: procedure_code field in procedure_type table is too small for valueset oids
### Describe the bug

some of the `OIDs` are longer than 31 chars or the code plus the `:` and code set prefix

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #5351

### CODE REVIEW

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV

File: sql/6_1_0-to-7_0_0_upgrade.sql [hunk 1]
```diff
@@ -671,3 +671,7 @@ ALTER TABLE `insurance_companies` ADD `cqm_sop` int DEFAULT NULL COMMENT 'HL7 So
 #IfNotRow2D list_options list_id order_type option_id order
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('order_type','order','Order',90,0);
 #EndIf
+
+#IfNotColumnType procedure_type procedure_code varchar(63)
+ALTER TABLE `procedure_type` MODIFY `procedure_code` varchar(63) NOT NULL DEFAULT '' COMMENT 'code identifying this procedure',
```
Comment (sjpadgett) [inline sql/6_1_0-to-7_0_0_upgrade.sql:676]: varchar(64) in keeping with size I changed procedure_order_code (L-530) procedure_code. 
Nitpicking I guess and won't matter in the end

**Review by sjpadgett — APPROVED** [review]
Thanks for taking care of our users Stephen!

File: sql/6_1_0-to-7_0_0_upgrade.sql [hunk 2]
```diff
@@ -671,3 +671,7 @@ ALTER TABLE `insurance_companies` ADD `cqm_sop` int DEFAULT NULL COMMENT 'HL7 So
 #IfNotRow2D list_options list_id order_type option_id order
 INSERT INTO list_options ( list_id, option_id, title, seq, is_default ) VALUES ('order_type','order','Order',90,0);
 #EndIf
+
+#IfNotColumnType procedure_type procedure_code varchar(63)
```
[the remark this answers is not in the document]
Comment (stephenwaite) [reply sql/6_1_0-to-7_0_0_upgrade.sql:675]: :sweat_smile:

**Pull request merged**